resources/settings.xml

```diff
@@ -2,7 +2,7 @@
 <settings>
   <category label="10080">
     <!-- Subtitles -->
-    <setting id="subtitles_enabled" type="bool" label="10093" default="false" />
+    <setting id="subtitles_enabled" type="bool" label="10093" default="true" />
   </category>
```

Comment:
If you're going to enable by default, just forget about the configuration setting I think